docs/README.md

```diff
@@ -40,6 +40,28 @@ To start the [docz](https://www.docz.site/) development server:
 npm start
 ```
 
+### Installing packages
+
+Atlantis are installed and updated using [npm](https://www.npmjs.com/). This following list
+has installation links for each package:
+
+#### Design system
+
+These are the core packages you'll need to build with Atlantis:
+
+- [Components](https://atlantis.getjobber.com/packages/components#installation)
```

Comment:
these links can be just `/packages/...`. This way they will work locally too

Reply:
I would also say lets link these to the page and not the `installation` heading, This way the reader will understand the package as well